components/cdc/src/endpoint.rs

```diff
@@ -583,7 +584,7 @@ impl<T: 'static + RaftStoreRouter<RocksEngine>> Endpoint<T> {
 
     fn register_min_ts_event(&self) {
         let timeout = self.timer.delay(self.min_ts_interval);
-        let tso = self.pd_client.get_tso();
+        let tso = Compat::new(self.pd_client.get_tso());
```

Comment:
Why not `get_tso().compact()`?

Reply:
compact() is a function dedicated to convert from 0.1 to 0.3. There is no trait which converts reversely. FYI https://docs.rs/futures/0.3.5/futures/compat/index.html